roles/KubevirtTemplateValidator/tasks/main.yml

```diff
@@ -10,6 +10,49 @@
     definition: "{{ item | from_yaml }}"
   with_items: "{{ lookup('template', 'service.yaml.j2').split('\n---\n') | select('search', '(^|\n)[^#]') | list }}"
   register: tv
+# Actively inject owner references in order to adopt existing resources during an upgrade
```

Comment:
And same here..

Reply:
Same here

Reply:
Same objection to the duplication based on kind. In fact, the `results[0]` based indexing assumes a lot about the source file.

Reply:
I don't see any other way to do it other than fetching the resources I want to own from the cluster one by one and patching them

Reply:
The issue with tv.results[0].result is that the code will break when someone changes the order of objects in service.yaml.j2. Why not simply iterate over all of them?

Reply:
The issue is that the file service.yaml.j2 contains cluster-scoped resources that cannot be owned by a namespaced object (ClusterRole, ClusterRoleBinding)